specification/network/resource-manager/Microsoft.Network/stable/2019-04-01/privateLinkService.json

```diff
@@ -422,6 +422,90 @@
           "final-state-via": "location"
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Network/locations/{location}/checkPrivateLinkServiceVisibility": {
+      "post": {
+        "tags": [
+          "PrivateLinkServices"
+        ],
+        "operationId": "CheckPrivateLinkServiceVisibility",
```

Comment:
Since this method is related to privateLinkService, maybe it would be better to name it `PrivateLinkServices_CheckPrivateLinkServiceVisibility`

Reply:
Fixed.